eks/vpn.tf

```diff
@@ -0,0 +1,46 @@
+resource "aws_eip" "vpn_eip" {
+  instance = aws_instance.vpn.id
+  vpc = true
+}
+
+resource "aws_instance" "vpn" {
+  ami = "ami-049aea444f70407b8"
```

Comment:
Can you make this a data.ami search by name? We want to be able to find the version across different regions and this will likely only return in one region.